test/unit/FormulateForm.test.js

```diff
@@ -43,7 +51,8 @@ describe('FormulateForm', () => {
     expect(wrapper.vm.registry.keys()).toEqual(['subinput1', 'subinput2'])
   })
 
-  it('deregisters a subcomponents', async () => {
+  // Vue-test-utils no longer provides `setData`
```

Comment:
`vue-test-utils` is more strongly recommending the avoidance of testing implementation details. Them removing `setData` is part of this.